inst/rmarkdown/templates/elsevier_article/resources/template.tex

```diff
@@ -105,7 +105,7 @@
             bookmarks=true,
             pdfauthor={$author-meta$},
             pdftitle={$title-meta$},
-            colorlinks=true,
+            colorlinks=$if(colorlinks)$$colorlinks$$else$true$endif$,
```

Comment:
I'm confused: if `colorlinks=false`, it seems that it will turn on coloring? How about

```
$if(colorlinks)$colorlinks=true,$endif$
```

?

I know this will change the default behavior of this template in previous versions (default will be "off" instead of "on"), but if users want to turn on coloring, they can set `colorlinks: true` in YAML.

Reply:
Hi, Yihui. I may have not understood how to code this properly. What I wanted to achieve was to avoid the hardcoded ```colorlink=true```in the template, and be able to pass ```true```or ```false```from yaml. For my own use (and perhaps most others) it would almost never make any sense to color links in a paper for submission to a journal, so I would be fine with the default corresponding to the links being rendered black (both urls and crossreferences).

Reply:
Okay, I think we are in agreement. Let's change this line to `colorlinks=$if(colorlinks)$true$else$false$endif$,`, which means the default will be black.